docs/oscar_references.bib

```diff
@@ -318,3 +318,42 @@ @inproceedings{BDLP19
   HAL_ID = {hal-02912148},
   HAL_VERSION = {v1},
 }
+
+@Book{DK98,
```

Comment:
Why DK98? This should be DK15 or DK2015?

Reply:
o.k.